ui/lib/UpdateStream.tsx

```diff
@@ -0,0 +1,7 @@
+import * as React from "react";
+
+const UpdateStream = () => {
```

Comment:
it'd be super helpful if this was implemented
Reason: You can't really delete streams. You can cancel them, but they don't go away. It'd be nice to detect an existing stream in the App and have the modal prompt you to update it (if it was canceled) rather than create another one

Reply:
The security properties are that a stream that ran out of funding is the same as being cancelled